python-api/src/main/python/livy/job_handle.py

```diff
@@ -247,8 +235,8 @@ def start(self):
             if not self.is_running and not self.stop_called:
                 self._timer = Timer(self.interval, self._run)
                 self._timer.start()
-                self.interval = min(self.interval * 2,
-                                    JobHandle._JOB_MAX_POLL_INTERVAL)
+                self.interval = min(
+                    self.interval * 2, JobHandle._JOB_MAX_POLL_INTERVAL)
```

Comment:
I think you're still confused about align vs. indent. `self.interval * 2` can live in the previous line like before.

This is wrong:

```
self.interval = min(self.interval * 2,
                    JobHandle._JOB_MAX_POLL_INTERVAL)
```

Because the second line is *aligned*. The following is right, because it's indented:

```
self.interval = min(self.interval * 2,
    JobHandle._JOB_MAX_POLL_INTERVAL)
```